android/src/main/java/com/applovin/reactnative/AppLovinMAXNativeAdView.java

```diff
@@ -108,19 +116,84 @@ public void setCustomData(@Nullable final String value)
         customData = value;
     }
 
-    public void setExtraParameters(@Nullable final ReadableMap readableMap)
+    public void setExtraParameters(@Nullable final ReadableArray readableArray)
```

Comment:
We should name parameters to describe what it is, not its type. So I'd name it `extraParameters`.